### DISCUSSION TITLE
Portable build for Windows?

### DISCUSSION THREAD

**Discussion by bickford-atherton-morrow** [vkbo/novelWriter#1411] [Ideas]
Just wondering if it might be possible to have a portable build for Windows?

**Comment by vkbo** [comment]
It's pure Python code, so novelWriter is portable directly from its source code. That's actually how I run it myself when I do my writing. The problem with Windows is that it doesn't have a native Python environment, so you need to install one to run the code on any given machine. 

For the current Windows releases, the environment is embedded in the install folder alongside the app. The install folder is therefore itself portable. You just need to make a custom launcher for it that calls `pythonw.exe novelWriter.py` in that folder.
